requirements.txt

```diff
@@ -1,2 +1,3 @@
 flask
```

Comment:
You can set the versions for all packages here, but not mandatory.

Reply:
I think it would be a good thing. Try to use specific versions instead of minimum versions to be sure all the packages interact as intended. Incompatibilities could appear with new package versions.